### ISSUE TITLE
vugufmt breaks on case sensitive input (vgform:Input, vgform:Select)

### ISSUE THREAD

**Issue by seanrmurphy** [vugu/vugu#154]
**Describe the bug**
vugufmt does not handle `vgform:Input` and `vgform:Select` properly - the resulting output loses the upper case letter after the colon.

**Software Versions**
Vugu version: v0.3.2-5-g1845706
Go version: go version go1.14.6 linux/amd64
Browser and version: Not relevant.

**To Reproduce**
Create a `.vugu` file which contains a `vgform:Input` element and run it through `vugufmt`

```
<div>
        <form>
                <div>
                        <label>Input Data:</label>
                        <vgform:Input type="string" :value='vgform.StringPtr{&c.InputString}'
                                ></vgform:Input>
                </div>

        </form>
</div>


<script type="application/x-go">

import "github.com/vugu/vugu/vgform"

type InputDataExample struct {

	InputString string
}

</script>
```

**Expected behavior**
Output:

```
/tmp> vugufmt input-data-example.vugu
<div>
	<form>
		<div>
			<label>Input Data:</label>
			<vgform:input type="string" :value='vgform.StringPtr{&c.InputString}'
                                ></vgform:input>
		</div>

	</form>
</div>


<script type="application/x-go">

import "github.com/vugu/vugu/vgform"

type InputDataExample struct {
	InputString string
}

</script>
```

`vgform:Input` should not have been modified to `vgform:input`

**Additional Notes**
None.

**Comment by seanrmurphy** [comment]
Also impacts `:Value` and `:Options` - should have flagged this in the above.

**Comment by bradleypeabody** [comment]
Thanks for this.  Yeah the vugufmt code is a bit out of date, needs some TLC.  I'll leave this open to address the next time maintenance is done on that tool.

**Comment by owenwaller** [comment]
Closing this as the problem seems to be resolved by #195.
